src/init/SeededPodInitializer.ts

```diff
@@ -0,0 +1,55 @@
+import type { RegistrationManager } from '../identity/interaction/email-password/util/RegistrationManager';
+import { getLoggerFor } from '../logging/LogUtil';
+import type { KeyValueStorage } from '../storage/keyvalue/KeyValueStorage';
+import { Initializer } from './Initializer';
+
+export interface SeederPodInitializerArgs {
+  /**
+   * Used for registering seeded pods.
+   */
+  registrationManager: RegistrationManager;
+  /**
+   * Settings that the seeded Pods should be created with.
+   */
+  configStorage: KeyValueStorage<string, unknown>;
+}
+
+/**
+ * Uses a {@link RegistrationManager} to initializes accounts and pods
+ * for all seeded pods. Reads the pod settings from seededPodConfigJson.
+ */
+export class SeededPodInitializer extends Initializer {
+  protected readonly logger = getLoggerFor(this);
+
+  private readonly registrationManager: RegistrationManager;
+  private readonly configStorage: KeyValueStorage<string, unknown>;
+
+  public constructor(args: SeederPodInitializerArgs) {
+    super();
+    this.registrationManager = args.registrationManager;
+    this.configStorage = args.configStorage;
+  }
+
+  public async handle(): Promise<void> {
+    let count = 0;
+    for await (const [ , value ] of this.configStorage.entries()) {
+      const config = value as NodeJS.Dict<unknown>;
+      config.confirmPassword = config.password;
+      config.createPod = true;
+      config.createWebId = true;
+      config.register = true;
```

Comment:
```suggestion
      const input = value as NodeJS.Dict<unknown>;
      const config = {
        confirmPassword: input.password,
        createPod: true,
        createWebId: true,
        register: true,
        ...input
      }
```
This way it is also possible for people to create a seeded pod config that only creates pods without accounts, or the opposite, if they set the correct booleans.